frontend/src/bundles/video_editor/components/timeline/subcomponents/item.tsx

```diff
@@ -0,0 +1,41 @@
+import { type Span } from 'dnd-timeline';
+import { useItem } from 'dnd-timeline';
+
+interface ItemProperties {
+	id: string;
+	span: Span;
+	children: React.ReactNode;
+}
+const Item = (properties: ItemProperties): JSX.Element => {
+    const { setNodeRef, attributes, listeners, itemStyle, itemContentStyle } =
+        useItem({
+            id: properties.id,
+            span: properties.span,
+        });
+
+    return (
+        <div ref={setNodeRef} style={itemStyle} {...listeners} {...attributes}>
+            <div style={itemContentStyle}>
+                <div
+                    style={{
+                        width: '100%',
+                        overflow: 'hidden',
+                        margin: '2px',
+                        borderRadius: '15px',
+                        backgroundColor: 'lightblue',
+                        color: 'darkblue',
+                        textAlign: 'center',
+                        display: 'flex',
+                        justifyContent: 'center',
+                        alignItems: 'center',
+                        height: '100%', 
+                    }}
+                >
+                    {properties.children}
+                </div>
+            </div>
+        </div>
```

Comment:
I guess since we use chakra ui we should avoid using html tags directly. Please review your components and replace them with chakra ui components (where it is possible). For example, you can replace div with a Box component from chakra ui. Make sure to import chakra ui components from ~/bundles/common/components/components.js.

Reply:
Thank you. Changed all html to chakra